From 356f4b2fea6e5e5d58f3d7e7a41441f4e768b514 Mon Sep 17 00:00:00 2001
From: Jamie Liu <jliu@nixprime.com>
Date: Fri, 11 Aug 2017 20:25:58 -0700
Subject: [PATCH 1/2] Disable arrows

---
 meta/feature/ligation.ptl | 163 +---------------------------------------------
 1 file changed, 1 insertion(+), 162 deletions(-)

diff --git a/meta/feature/ligation.ptl b/meta/feature/ligation.ptl
index af311eca4457..fcb222d5e2e3 100644
--- a/meta/feature/ligation.ptl
+++ b/meta/feature/ligation.ptl
@@ -245,168 +245,7 @@ export : define [buildLigations chain-rule lookupOrder commonList features looku
 					inputIndex 0
 
 	do "Arrows" 
-		includeLookup
-			.type 'gsub_chaining'
-			.subtables : list
-				chain-rule # <---->, <====>
-					less       ~> preserved
-					arrowStick ~> [lsx 'lc']
-					arrowStick ~> [lsx 'cc']
-					arrowStick ~> [lsx 'cc']
-					arrowStick ~> [lsx 'cr']
-					greater    ~> preserved
-				chain-rule # <--->, <===>
-					less       ~> preserved
-					arrowStick ~> [lsx 'lc']
-					arrowStick ~> [lsx 'cc']
-					arrowStick ~> [lsx 'cr']
-					greater    ~> preserved
-				chain-rule # <-->, <==>
-					less       ~> preserved
-					arrowStick ~> [lsx 'lc']
-					arrowStick ~> [lsx 'cr']
-					greater    ~> preserved
-				chain-rule # <->, <=>
-					less       ~> preserved
-					arrowStick ~> [lsx 'lr']
-					greater    ~> preserved
-				chain-rule # <--<, <==<
-					less       ~> preserved
-					arrowStick ~> [lsx 'lc']
-					arrowStick ~> [lsx 'cj']
-					less       ~> preserved
-				chain-rule # <-<, <=<
-					less       ~> preserved
-					arrowStick ~> [lsx 'lj']
-					less       ~> preserved
-				chain-rule # <---, <===
-					less       ~> preserved
-					arrowStick ~> [lsx 'lc']
-					arrowStick ~> [lsx 'cc']
-					arrowStick ~> [lsx 'cf']
-				chain-rule # <--, <==
-					less       ~> preserved
-					arrowStick ~> [lsx 'lc']
-					arrowStick ~> [lsx 'cf']
-				chain-rule # <<-, <<=
-					less       ~> preserved
-					less       ~> [only 'arrow2' : lsx 'shift1']
-					arrowStick ~> [only 'arrow2' [lsx 'l1f'] : lsx 'lf']
-				chain-rule # <-
-					less       ~> preserved
-					hyphen     ~> [lsx 'lf']
-				chain-rule # <=
-					less       ~> preserved
-					equal      ~> [only 'arrowZALE' : lsx 'lf']
-				chain-rule # <!---
-					less       ~> preserved
-					exclam     ~> [lsx 'htmlcommentstart']
-					hyphen     ~> [lsx 'lxc']
-					hyphen     ~> [lsx 'cc']
-					hyphen     ~> [lsx 'cf']
-				chain-rule # <!--
-					less       ~> preserved
-					exclam     ~> [lsx 'htmlcommentstart']
-					hyphen     ~> [lsx 'lxc']
-					hyphen     ~> [lsx 'cf']
-				chain-rule # >>-, >>=
-					greater    ~> preserved
-					greater    ~> [only 'arrow2' : lsx 'shiftN1']
-					arrowStick ~> [only 'arrow2' : lsx 'j1f']
-				chain-rule # >-->, >==>
-					greater    ~> preserved
-					arrowStick ~> [lsx 'jc']
-					arrowStick ~> [lsx 'cr']
-					greater    ~> preserved
-				chain-rule # >->, >=>
-					greater    ~> preserved
-					arrowStick ~> [lsx 'jr']
-					greater    ~> preserved
-				chain-rule # >-
-					greater    ~> preserved
-					hyphen     ~> [only 'arrow2' : lsx 'jf']
-				chain-rule # --->
-					arrowStick ~> [lsx 'fc']
-					arrowStick ~> [lsx 'cc']
-					arrowStick ~> [lsx 'cr']
-					greater    ~> preserved
-				chain-rule # -->
-					arrowStick ~> [lsx 'fc']
-					arrowStick ~> [lsx 'cr']
-					greater    ~> preserved
-				chain-rule # ->-, =>=
-					arrowStick ~> [only 'arrow2' : lsx 'fr']
-					greater    ~> preserved
-					arrowStick ~> [only 'arrow2' : lsx 'jf']
-				chain-rule # ->>, =>>
-					arrowStick ~> [only 'arrow2' [lsx 'fr1'] : lsx 'fr']
-					greater    ~> [only 'arrow2' : lsx 'shift1']
-					greater    ~> preserved
-				chain-rule # ->, =>
-					arrowStick ~> [lsx 'fr']
-					greater    ~> preserved
-				chain-rule # -<-, =<=
-					arrowStick ~> [only 'arrow2' : lsx 'fj']
-					less       ~> preserved
-					arrowStick ~> [only 'arrow2' : lsx 'lf']
-				chain-rule # -<<, =<<
-					arrowStick ~> [only 'arrow2' : lsx 'fj1']
-					less       ~> [only 'arrow2' : lsx 'shiftN1']
-					less       ~> preserved
-				chain-rule # -<, =<
-					arrowStick ~> [only 'arrow2' : lsx 'fj']
-					less       ~> preserved
-				chain-rule # >=
-					greater    ~> preserved
-					equal      ~> [only 'arrowZAGE' : lsx 'jf']
-		# <<, >>, <<<, >>>
-		includeLookup
-			.type 'gsub_chaining'
-			.subtables : list
-				chain-rule # <<<
-					less       ~> [only 'arrow2' : lsx 'shiftN0t']
-					less       ~> preserved
-					less       ~> [only 'arrow2' : lsx 'shift0t']
-				chain-rule # >>>
-					greater    ~> [only 'arrow2' : lsx 'shift0t']
-					greater    ~> preserved
-					greater    ~> [only 'arrow2' : lsx 'shiftN0t']
-				chain-rule # <<
-					less       ~> [only 'arrow2' : lsx 'shiftN0t']
-					less       ~> [only 'arrow2' : lsx 'shift0t']
-				chain-rule # >>
-					greater    ~> [only 'arrow2' : lsx 'shift0t']
-					greater    ~> [only 'arrow2' : lsx 'shiftN0t']
-		# Extended arrow sticks
-		# --------------------->
-		includeLookup
-			.type 'gsub_reverse'
-			.subtables : list 
-				object
-					match      {{'hyphen' 'equal'} [stickClass 'fr']}
-					to         [stickClass 'fz']
-					inputIndex 0
-				object
-					match      {{'hyphen' 'equal'} [stickClass 'fc']}
-					to         [stickClass 'fz']
-					inputIndex 0
-				object
-					match      {{'hyphen' 'equal'} [stickClass 'fz']}
-					to         [stickClass 'fz']
-					inputIndex 0
-		# <--------------------, <------------------->
-		includeLookup
-			.type 'gsub_chaining'
-			.subtables : list
-				chain-rule ([stickClass 'cf'] ~> [stickClass 'cc']) ([stickClass 'fc'] ~> [stickClass 'cc'])
-				chain-rule ([stickClass 'zf'] ~> [stickClass 'zc']) ([stickClass 'fr'] ~> [stickClass 'cr'])
-				chain-rule ([stickClass 'zf'] ~> [stickClass 'zc']) ([stickClass 'fc'] ~> [stickClass 'cc'])
-				chain-rule ([stickClass 'lf'] ~> [stickClass 'lc']) ([stickClass 'fz'] ~> [stickClass 'cz'])
-				chain-rule ([stickClass 'cf'] ~> [stickClass 'cc']) ([stickClass 'fz'] ~> [stickClass 'cz'])
-				chain-rule ([stickClass 'zf'] ~> [stickClass 'zc']) ([stickClass 'fz'] ~> [stickClass 'cz'])
-				chain-rule [stickClass 'lf'] [stick 'zf']
-				chain-rule [stickClass 'cf'] [stick 'zf']
-				chain-rule [stickClass 'zf'] [stick 'zf']
+
 	# /\ and \/
 	do "logical" : includeLookup
 		.type 'gsub_chaining'
-- 
2.7.4

